website/client/components/tasks/task.vue

```diff
@@ -598,13 +598,13 @@ export default {
       return false;
     },
     controlClass () {
-      return this.getTaskClasses(this.task, 'control', this.dueDate);
+      return this.getTaskClasses(this.task, 'control', this.dueDate, this.$route.name);
```

Comment:
I think this `this.$route.name` should come from a prop of `task.vue` itself so that its not based on the route